.ci/AML-nightly-gpu.yml

```diff
@@ -37,20 +18,147 @@ pr:
 - master
 
 jobs:
-- job: Smoke
-  displayName : 'Smoke: no CV tests yet'
+- job: SmokeCPU
+  displayName : 'SmokeCPU'
   pool:
     vmImage: 'ubuntu-16.04'
 
   variables:
+  - group: AzureMLConfig
```

Comment:
can these be parameterized with yaml templating or something? there is a lot of duplicate code with the variables. It would be a bit troublesome to make change since we'd have to change all of them.

Reply:
No, many of those parameters are specific to the test.  For those that are not specific, we decided not to put them in a separate template as it would obfuscate the content and we decided clarity was more important.  Typically, once tests are running regularly, it's not the norm to require any changes.